conf.d/electrum

```diff
@@ -1,22 +1,35 @@
 #!/bin/bash -e
 
 SRC=/usr/local/src
-PATCHES=$SRC/electrum-patches
+PATCHES=$SRC/electrum.patch
 
 # install electrum deps
 easy_install $SRC/ecdsa-*.tar.gz
 easy_install $SRC/slowaes-*.tar.gz
-rm $SRC/{ecdsa-*.tar.gz,slowaes-*.tar.gz}
+easy_install $SRC/jsonrpclib-*.tar.gz
```

Comment:
bitaddress.org-3.2.0.zip comes from www.bitaddress.org . A PGP sig is available on site

Electrum-2.6.4.tar.gz obviously comes from electrum.org . A PGP sig is available on site

All others come from pypi.python.org:
colorama-0.3.7.zip comes from https://pypi.python.org/pypi/colorama/0.3.7
dnspython-1.12.0.zip comes from https://pypi.python.org/pypi/dnspython/1.12.0
jsonrpclib-0.1.7.tar.gz comes from https://pypi.python.org/pypi/jsonrpclib/0.1.7
pbkdf2-1.3.tar.gz comes from https://pypi.python.org/pypi/pbkdf2/1.3
protobuf-3.0.0b2.post2.zip comes from https://pypi.python.org/pypi/protobuf/3.0.0b2
qrcode-5.2.2.tar.gz comes from https://pypi.python.org/pypi/qrcode/5.2.2
requests-2.9.1.tar.gz comes from https://pypi.python.org/pypi/requests/2.9.1
six-1.10.0.tar.gz comes from https://pypi.python.org/pypi/six/1.10.0

The next 3 were not touched but probably also come from pypi.python.org:
ecdsa-0.10.tar.gz
python-zxcvbn_bc1c2d3.tar.gz
slowaes-0.1a1.tar.gz

I noticed that I forgot to remove the folder/file electrum-patches (replaced by a single patch file) and the file pbkdf2-1.3.tar.gz.1 (duplicate). I removed them in another commit in my fork.